bundles/CoreBundle/Migrations/Version20210702102100.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Pimcore\Tool\Console;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\Process;
+
+final class Version20210608094532 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Updates class definition files';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->rebuildClassesCommand();
+    }
+
+    public function down(Schema $schema): void
+    {
+        $this->rebuildClassesCommand();
+    }
+
+    /**
+     * @throws \Exception|ProcessFailedException
+     */
+    private function rebuildClassesCommand()
+    {
+        $cmd = [
```

Comment:
rebuild command still saves the class and updates the db. As said before, we just need to regenerate the class files here.